data/governance.yml

```diff
@@ -0,0 +1,508 @@
+teams:
+- id: compiler
+  name: Compiler
+  description: The OCaml Compiler team, responsible for the development and maintenance
+    of the language, the standard library and the compiler tools.
+  contacts:
+  - name: Mailing List
+    link: https://sympa.inria.fr/sympa/subscribe/caml-list
+    kind: email
+  - name: Discord
+    link: https://discord.com/channels/436568060288172042/904141760052228116
```

Comment:
For the compiler, I think that the the discord link should be removed, since the channel is not official in any way.
Keeping the link to the `caml-list` is debatable, but it is probably a good idea to keep an email channel.

Reply:
Done!